components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -184,12 +263,38 @@
   &--level-1 {
     font: var(--font-style-nav-primary-1);
     color: var(--color-heading);
-    padding: var(--size-spacing-2) var(--size-spacing-5);
     margin-bottom: var(--size-spacing-3);
 
+    [data-menu-variation='basic'] & {
+      padding: var(--size-spacing-2) var(--size-spacing-6) var(--size-spacing-4);
+      position: relative;
+
+      // basic menu items have a bottom border
+      &::after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        left: var(--size-spacing-5);
+        right: var(--size-spacing-5);
+        height: var(--border-thickness-1);
+        background-color: var(--color-gray-200);
+      }
+    }
+
+    [data-menu-variation='mega'] & {
+      padding: var(--size-spacing-2) var(--size-spacing-5);
+    }
+
     .primary-nav__item--level-1:first-child & {
       padding-top: 0;
     }
+
+    // remove last child's bottom border
+    [data-menu-variation='basic'] .primary-nav__item--level-1:last-child & {
+      &::after {
+        content: unset;
+      }
+    }
```

Comment:
Based on my refactor suggestion above, let's use this:
```
    [data-menu-variation='basic'] :last-child > &::after {
      content: unset;
    }
```

And while you're at it, you can update the code right before this to match:
```
    :first-child > & {
      padding-top: 0;
    }
```

I like that pattern better because it's more generic (in case the class name changed... unlikely, but possible) but has the same level of precision, because of the "immediate child" selector (`>`).

Reply:
This should be fixed now.